---
layout: page
permalink: /publications/index.html
title: Publications
---

## Preprint


## Selected publications 
See my full Publication at <a href="https://scholar.google.com/citations?hl=en&user=Yis2IzgAAAAJ">Google Scholar</a> <br><br>
(^ Correspondence author) <br>
<ul>
<li><p>Chunqiu Xia, Shi-Hao Feng, Ying Xia, <b>Xiaoyong Pan</b>^, Hong-Bin Shen^. Leveraging Scaffold Information to Predict Protein-ligand Binding Affinity with an Empirical Graph Neural Network. Briefings in Bioinformatics, 2022(in press) [<a href="http://www.csbio.sjtu.edu.cn/bioinf/EGNA/">Server</a>]</p>
	</li>
	<li>
	 <p>Ying Xia, Chun-Qiu Xia, <b>Xiaoyong Pan</b>^, Hong-Bin Shen^. BindWeb: a web server for ligand binding residue and pocket prediction from protein structures. Protein Science, 2022 (in press).[<a href="http://www.csbio.sjtu.edu.cn/bioinf/BindWeb/">Server</a>]
		 </p>
  </li>	
<li>
    <p>Jiachen Li, Siheng Chen, <b>Xiaoyong Pan</b>, Ye Yuan, Hong-Bin Shen.Cell clustering for spatial transcriptomics data with graph neural networks. Nature Computational Science, 2022 (in press). </p>
  </li>	
	
<li><p>	Qunzhuo Wu, Zhaohong Deng, <b>Xiaoyong Pan</b>, Hong-Bin Shen, Kup-Sze Choi, Shitong Wang, Jing Wu, and Dong-Jun Yu. MDGF-MCEC: A multi-view dual attention embedding model with cooperative ensemble learning for circRNA-disease association prediction. Briefings in Bioinformatics, 2022 (in press).</p>
	 </li>	
	
<li>
    <p>Yong-Xian Fan^, Guicong Sun, <b>Xiaoyong Pan^</b>.ELMo4m6A: a contextual language embedding-based predictor for detecting RNA N6-methyladenosine sites. IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2022 (in press). [<a href="https://github.com/GuicongSun/ELMo4m6A">Code</a>]</p>
  </li>	
<li><p> 
	Chunqiu Xia, Shi-Hao Feng, Ying Xia, <b>Xiaoyong Pan^</b>, and Hong-Bin Shen^. Fast protein structure comparison through effective representation learning with contrastive graph neural networks.
PLoS Computational Biology, 2022 (in press).  [<a href="http://www.csbio.sjtu.edu.cn/bioinf/GraSR/">Server</a>]
	</p> </li>
	<li><p> 
Yong-Xian Fan^, Meijun Chen, <b>Xiaoyong Pan^</b>. GCRFLDA: Scoring lncRNA-disease associations using graph convolution matrix completion with conditional random field.
Briefings in Bioinformatics, 2021 (in press). 
	</p> </li>
	<li><p> 
Hehe Wu, <b>Xiaoyong Pan^</b>, Yang Yang and Hong-Bin Shen^. Recognizing binding sites of poorly characterized RNA-binding proteins on circular RNAs using attention Siamese network.
Briefings in Bioinformatics, 2021 (in press).  [<a href="https://github.com/hehew321/iDeepC">Code</a>]
	</p> </li>
<li><p> 	
	Hui Li, Zhaohong Deng, Haitao Yang, <b>Xiaoyong Pan</b>, Zhisheng Wei, Hong-Bin Shen, Kup Sze Choi, Lei Wang, Shitong Wang, and Jing Wu. circRNA-binding protein sites prediction based on multi-view deep learning, subspace learning and multi-view classifier. Briefings in Bioinformatics, 2021 (in press).  [<a href=" https://github.com/Rebecca3150/DMSK">Code</a>]
	</p> </li>
	
<li><p> 
Cheng-Peng Zhou, Di Wang, <b>Xiaoyong Pan</b>, and Hong-Bin Shen. Protein structure refinement using multi-objective particle swarm optimization with decomposition strategy.
International Journal of Molecular Sciences, 2021 (in press).  [<a href="http://www.csbio.sjtu.edu.cn/bioinf/AIR/">Code</a>]
	</p> </li>
<li>
    <p> <b>Xiaoyong Pan^</b>, Lei Chen, Ｍin Liu, Zhibin Niu, Tao Huang, Yu-Dong Cai. <a href="">Identifying protein subcellular locations with embeddings-based node2loc.</a> IEEE/ACM Transactions on Computational Biology and Bioinformatics, 2021, in press. [<a href="https://github.com/xypan1232/node2loc">Code</a>]</p>
  </li>	
  
<li>
    <p> Yang Lin, <b>Xiaoyong Pan^</b>, and Hong-Bin Shen^. <a href="">lncLocator 2.0: a cell-line-specific subcellular localization predictor for long non-coding RNAs with interpretable deep learning</a>. Bioinformatics, 2021, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/lncLocator2/">Server</a>]</p>
  </li>	
<li>
    <p> Ying Xia, Chunqiu Xia, <b>Xiaoyong Pan^</b>, and Hong-Bin Shen^. <a href="">GraphBind: protein structural context embedded rules learned by hierarchical graph neural networks for recognizing nucleic-acid-binding residues</a>. Nucleic Acids Research, 2021, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/GraphBind/">Server</a>]</p>
  </li>	

<li>
    <p><b>Xiaoyong Pan^</b>, Yi Fang, Xianfeng Li, Yang Yang, and Hong-Bin Shen. <a href="">RBPsuite: RNA-protein binding sites prediction suite based on deep learning</a>. BMC Genomics, 2020, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/RBPsuite/">Server</a>]</p>
  </li>	
	
<li>
    <p><b>Xiaoyong Pan^</b>, Jasper Zuallaert, Xi Wang, Hong-Bin Shen, Elda Posada Campos, Denys O. Marushchak, Wesley De Neve. <a href="">ToxDL: Deep learning using primary structure and domain embeddings for assessing protein toxicity</a>. Bioinformatics, 2020, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/ToxDL">Server</a>]  [<a href="https://github.com/xypan1232/ToxDL">Code</a>]</p>
  </li>	
<li>
	<p>Sikai Ling, Shiqi Yang, Xinde Hu, Di Yin, Yao Dai, Xiaoqing Qian, Dawei Wang, <b>Xiaoyong Pan</b>, Jiaxu Hong, Xiaodong Sun, Hui Yang, Soren Riis Paludan and Yujia Cai. <a href="https://www.nature.com/articles/s41551-020-00656-y">Lentiviral delivery of co-packaged Cas9 mRNA and a Vegfa-targeting guide RNA prevents wet age-related macular degeneration in mice</a>. Nature Biomedical Engineering, 2020, in press. </p>
</li>	
<li>
    <p>Haitao Yang, Zhaohong Deng, <b>Xiaoyong Pan</b>, Hong-bin Shen, Kup-Sze, Choi, Lei Wang, Shitong Wang, Jing Wu. <a href="">RNA-binding protein recognition based on multi-view deep feature and multi-label learning</a>. Briefings in Bioinformatics, 2020, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/iDeepMV/">Server</a>]  [<a href="http://github.com/uchihayht/iDeepMV">Code</a>]</p>
  </li>	

<li>
    <p>Wei-Xun Zhang, <b>Xiaoyong Pan</b>, Hong-Bin Shen. <a href="">Signal-3L 3.0: Improving signal peptide prediction through combining attention deep learning with window-based scoring</a>. Journal of Chemical Information and Modeling, 2020, In press. [<a href="http://www.csbio.sjtu.edu.cn/bioinf/Signal-3L/">Server</a>]</p>
  </li>	

<li>
    <p><b>Xiaoyong Pan^</b>, Hong-Bin Shen^. <a href="">Scoring disease-microRNA associations by integrating disease hierarchy into graph convolutional networks</a>. Pattern Recognition. 2020, In press. </p>
  </li>	
	
<li>
<p> Chun-Qiu Xia, <b>Xiaoyong Pan</b>, and Hong-Bin Shen. Protein-ligand binding residue prediction enhancement through hybrid deep heterogeneous learning of sequence and structure data. Bioinformatics, 2020 (in press). [<a href="http://www.csbio.sjtu.edu.cn/bioinf/delia/">Server</a>] </p>
 </li>
	
<li>
    <p><b>Xiaoyong Pan^</b>, Hong-Bin Shen^. <a href="https://www.cell.com/iscience/fulltext/S2589-0042(19)30351-7">Inferring disease-associated microRNAs using semi-supervised multi-label graph convolutional networks</a>. iScience. 20: 265-277. [<a href="https://github.com/xypan1232/DimiG">Code</a>]</p>
  </li>
	
<li>
<p>Kaiming Zhang*, <b>Xiaoyong Pan*</b>, Yang Yang, and Hong-Bin Shen. <a href="https://rnajournal.cshlp.org/content/early/2019/09/19/rna.070565.119.short?rss=1">CRIP: predicting circRNA-RBP interaction sites using a codon-based encoding and hybrid deep neural networks</a>. RNA, 2019 (in press) (shared first author). [<a href="https://github.com/kavin525zhang/CRIP">Code</a>]</p>
  </li>
<li>
    <p><b>Xiaoyong Pan</b>, Yang Yang, Chun-Qiu Xia, Aashiq H. Mirza, and Hong-Bin Shen. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/wrna.1544">Recent Methodology and Progress of Deep learning for RNA-protein interaction prediction</a>. WIREs RNA, 2019, in press.</p>		    
  </li>
  
<li>
    <p>	
	Jasper Zuallaert, <b>Xiaoyong Pan</b>, Yvan Saeys, Xi Wang and Wesley De Neve. <a href="https://drive.google.com/file/d/1n_59HKJR430N0aIpfv_44Yr9pxhhrYZs/view">Investigating the biological relevance in trained embedding representations of protein sequences</a>. ICML Workshop on Computational Biology 2019. </p>
  </li>
	    
<li>
    <p><b>Xiaoyong Pan</b>, Kai Xiong, Christian Anthon, Poul Hyttel, Kristine Freude, Lars Juhl Jensen, Jan Gorodkin. <a href="https://www.mdpi.com/2073-4425/9/11/536">WebCircRNA: Classifying the circular RNA potential of coding and 
noncoding RNA </a>. Genes. 2018, 9(11), 536. [<a href="https://rth.dk/resources/webcircrna">Webserver</a>]</p>
  </li>
	
<li>
    <p><b>Xiaoyong Pan</b>, Lars Juhl Jensen, Jan Gorodkin. <a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty859/5116144?redirectedFrom=fulltext">Inferring disease-associated long non-coding RNAs using genome-wide tissue expression profiles</a>. Bioinformatics. In press. [<a href="https://github.com/xypan1232/DislncRF">Code</a>]</p>
  </li>
	
<li>
    <p><b>Xiaoyong Pan</b>, XiaoHua Hu, Yu-Hang Zhang, Lei Chen, LiuCun Zhu, ShiBao Wan, Tao Huang, Yu-Dong Cai. <a href="https://link.springer.com/article/10.1007/s00438-018-1488-4">Identification of the Copy Number Variant Biomarkers for Breast Cancer Subtypes</a>. Molecular Genetics and Genomics. 2018, 294: 95–110 </p>
  </li>	
  
 <li>
    <p><b>Xiaoyong Pan^</b>, Yong-Xian Fan, Jue Jia, Hong-Bin Shen^. <a href="https://link.springer.com/article/10.1007/s11432-018-9558-2">Identifying RNA-binding proteins using multi-label deep learning</a>. SCIENCE CHINA Information Sciences. 2019, 62(1). [<a href="https://github.com/xypan1232/iDeepM">Code</a>]</p>
  </li> 

<li>
    <p><b>Xiaoyong Pan</b>, Anne Wenzel, Lars Juhl Jensen, Jan Gorodkin. <a href="https://doi.org/10.1371/journal.pone.0202369">Genome-Wide Identification of Clusters of Predicted MicroRNA Binding Sites as MicroRNA Sponge Candidates</a>. Plos One. 2018, 13(8): e0202369. [<a href="https://rth.dk/resources/mirnasponge/">Webserver</a>] </p>
  </li>

  
<li>
    <p><b>Xiaoyong Pan^</b>, Hong-Bin Shen^. <a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty364/4990826">Predicting RNA-protein binding sites and motifs through combining local and global deep convolutional neural networks</a>. Bioinformatics. 2018, 34: 3427–3436. [<a href="https://github.com/xypan1232/iDeepE">Code</a>]</p>
  </li>

<li>
    <p><b>Xiaoyong Pan^</b>, Peter Rijnbeek, Junchi Yan, Hong-Bin Shen^. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-018-4889-1">Prediction of RNA-protein sequence and structure binding preferences using deep convolutional and recurrent neural networks</a>. BMC Genomics, 2018, 19:511.[<a href="https://github.com/xypan1232/iDeepS">Code</a>]</p>
  </li>
  <li>
    <p>Zhen Cao*, <b>Xiaoyong Pan*</b>, Yang Yang*, Yan Huang, Hong-Bin Shen^. <a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/bty085/4860366?redirectedFrom=fulltext">The lncLocator: a subcellular localization predictor for long non-coding RNAs based on a stacked ensemble classifier</a>. Bioinformatics, 2018, 34(13):2185-2194 (shared first author). [<a href="www.csbio.sjtu.edu.cn/bioinf/lncLocator">Webserver</a>]</p>
  </li>
  
  <li>
    <p><b>Xiaoyong Pan^</b>, Hong-Bin Shen^. <a href="https://www.sciencedirect.com/science/article/pii/S0925231218304685">Learning distributed representations of RNA sequences and its application for predicting RNA-protein binding sites with a convolutional neural network</a>. Neurocomputing. 2018, 305: 51-58. [<a href="https://github.com/xypan1232/iDeepV">Code</a>]</p>
  </li>
  
  <li>
    <p>Lei Chen*, <b>Xiaoyong Pan*</b>, XiaoHua Hu*, Yu-Hang Zhang, ShaoPeng Wang, Tao Huang, Yu-Dong Cai. <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.31554">Gene expression differences among different MSI statuses in colorectal cancer</a>. International Journal of Cancer. In press (shared first author).</p>
  </li>
  <li>
  <p> Yong-Xian Fan, <b>Xiaoyong Pan</b>, Yang Zhang^ and Hong-Bin Shen^. LabCaS for ranking potential calpain substrate cleavage sites from amino acid sequence. Methods in Molecular Biology, In press [<a href="http://www.csbio.sjtu.edu.cn/bioinf/LabCaS">Webserver</a>]</p> 
  </li>
  <li>
    <p><b>Xiaoyong Pan^</b>, Junchi Yan. <a href="https://mlcb.github.io/">Attention based convolutional neural network for predicting RNA-protein binding sites</a>. NIPS Workshop on Machine Learning in Computational Biology, 2017. [<a href="https://github.com/xypan1232/iDeepA">Code</a>]</p>
  </li>	
  <li>
    <p>Lei Chen, Yu-Hang Zhang, Guohua Huang, <b>Xiaoyong Pan</b>, ShaoPeng Wang, Tao Huang, Yu-Dong Cai. <a href="https://www.ncbi.nlm.nih.gov/labs/articles/28913654">Discriminating circRNAs from other lncRNAs using a hierarchical extreme learning machine (H-ELM) algorithm with feature selection</a>. Molecular Genetics and Genomics. 2017 Sep 14. doi: 10.1007/s00438-017-1372-7.</p>
  </li>
  <li>
    <p><b>Xiaoyong Pan^</b>, Hong-Bin Shen^.  <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1561-8">RNA-protein binding motifs mining with a new hybrid deep learning based cross-domain knowledge integration approach </a>. BMC bioinformatics, 2017, 18(1):136. [<a href="https://github.com/xypan1232/iDeep">Code</a>]</p>
  </li>
  <li>
    <p>Alexander Junge, Jan C. Refsgaard, Christian Garde, <b>Xiaoyong Pan</b>, Alberto Santos, Ferhat Alkan, Christian Anthon, Christian von Mering, Christopher T. Workman, Lars Juhl Jensen, and Jan Gorodkin. 
	RAIN: RNA-protein Association and Interaction Networks. Database (Oxford), 2017, baw167.</p>
  </li>
  <li>
    <p><b>Xiaoyong Pan</b>, Yong-Xian Fan, Junchi Yan, Hong-Bin Shen. IPMiner: hidden ncRNA-protein interaction sequential pattern mining with stacked autoencoder for accurate computational prediction. BMC genomics, 2016, 17(1): 582.</p>
  </li>
  <li>
    <p><b>Xiaoyong Pan</b>, Hong-Bin Shen. OUGENE: a disease associated over-expressed and under-expressed gene database. Science Bulletin, 2016, 61(10):752-754.</p>
  </li> 
  <li>
    <p><b>Xiaoyong Pan^</b>, Kai Xiong. PredcircRNA: computational classification of circular RNA from other long non-coding RNA using hybrid features. Molecular BioSystems, 2015, 11(8): 2219-2226.</p>
  </li>
  <li>
    <p>Aashiq H Mirza, Claus HB Berthelsen, Stefan E Seemann, <b>Xiaoyong Pan</b>, Klaus S Frederiksen, Mogens Vilien, Jan Gorodkin, Flemming Pociot. Transcriptomic landscape of lncRNAs in inflammatory bowel disease. Genome medicine, 2015, 7(1):39.</p>
  </li> 
  <li>
    <p><b>Xiaoyong Pan^</b>, Lin Zhu, Yong-Xian Fan, Junchi Yan. Predicting protein–RNA interaction amino acids using random forest based on submodularity subset selection. Computational biology and chemistry, 2014, 53,:324-330.</p>
  </li>   
  <li>
    <p><b>Xiao-Yong Pan</b>, Ye Tian, Yan Huang, Hong-Bin Shen. Towards better accuracy for missing value estimation of epistatic miniarray profiling data by a novel ensemble approach. Genomics, 2011, 97(5): 257-264.</p>
  </li>  
  <li>
    <p>Ya-Nan Zhang, <b>Xiao-Yong Pan</b>, Yan Huang, Hong-Bin Shen. Adaptive compressive learning for prediction of protein–protein interactions from primary sequence. Journal of theoretical biology, 2011, 283(1): 44-52.</p>
  </li>  
  <li>
    <p><b>Xiao-Yong Pan</b>, Ya-Nan Zhang, Hong-Bin Shen. Large-Scale prediction of human protein− protein interactions from amino acid sequence based on latent topic features. Journal of Proteome Research, 2010, 9(10): 4992-5001.</p>
  </li>  
  <li>
    <p>Xue-Mei Ding, <b>Xiao-Yong Pan</b>, Chen Xu, Hong-Bin Shen. Computational prediction of DNA-protein interactions: a review. Current computer-aided drug design, 2010, 6(3): 197-206.</p>
  </li>  
  <li>
    <p><b>Xiao-Yong Pan</b>, Hong-Bin Shen. Robust prediction of B-factor profile from sequence using two-stage SVR based on random forest feature selection. Protein and peptide letters, 2009, 16 (12), 1447-1454.</p>
  </li>    

</ul>
<br><br>

## Poster and Technical Communications
<ul>
<li><p><b>Xiaoyong Pan</b>, Peter Rijnbeek. <a href="https://atlasofscience.org/a-novel-deep-learning-based-method-for-predicting-rna-protein-interactions/"> Hybrid deep learning for predicting hypertensive disorder onset using temporal and non-temporal data </a>. OHDSI Symposium, 2017. [Poster]</p>
</li>
</ul>
<ul>
<li><p>Peter Rijnbeek, <b>Xiaoyong Pan</b>, Jenna M. Reps. <a href="http://www.ohdsi.org/web/wiki/lib/exe/fetch.php?media=symposium_2017:ohdsi_symposium_2017_learningcurves.pdf">Do we really need ‘Big Data” for patient-level predictive modelling? </a>. OHDSI Symposium, 2017.[Poster]. <b>Best contribution award to OHDSI 2017</b></p>
</li>
</ul>
<ul>
<li><p><b>Xiaoyong Pan</b>, Hong-Bin Shen. <a href="https://atlasofscience.org/a-novel-deep-learning-based-method-for-predicting-rna-protein-interactions/"> A novel deep learning-based method for predicting RNA-protein interactions </a>. Atlas of Science, 2017.</p>
</li>
</ul>
<br><br>

## Grants
Be involved in the National Natural Science Foundation of China (No. 61462018，61762026)

## Professional Activities
<b>Reviewer </b> for Bioinformatics, Journal of Proteome Research, BBA - Molecular Basis of Disease, Computational biology and chemistry, IEEE Journal of Biomedical and Health Informatics, Interdisciplinary Sciences: Computational Life Sciences, IEEE Access, RSC Advances, Oncotarget, BioMed Research International, Molecules.<br>

IEEE Access Leading Guest editor (2017-) <br>

## Organizational activities
<ul><li>Leading guest Editor Special issue <a href="http://ieeeaccess.ieee.org/special-sections/sequential-data-modeling-emerging-applications/">Sequential Data Modeling and Its Emerging Applications </a> in IEEE Access. </li> </ul>
<ul><li> Organizers for <a href="http://vision.ia.ac.cn/CCPRWorkshop2018/index.html">Correspondence Problem in Computer Vision and Pattern Recognition </a> in ICPR 2018. </li>
	
</ul>


